File: Instructor Java/collections/Java-Array-Double/Java/ArrayDouble.java
import java.util.Random;

/**
 * ArrayDemo
 */
public class ArrayDouble {

  public static double[] rand(int size, double low, double high) {
    var ret = new double[size];
    var bound = high - low;
    var rand = new Random();
    for (var i = 0; i < size; i++) {
      ret[i] = low + bound * rand.nextDouble();
    }
    return ret;
  }

  public static void print(double[] arr) {
    var index = 0;
    for (var item : arr) {
      System.out.printf("index=%3d ==>  Value=%5f %n", index++, item);
    }
  }

}

File: Instructor Python/Collections/Python-Array-Double/Python/ArrayDouble.py
import random


class ArrayDouble(object):
    @staticmethod
    def rand(size, low, high):
        array = []
        for _ in range(0, size):
            array.append(random.uniform(low, high))
        return array

    @staticmethod
    def print(array):
        index = 0
        for item in array:
            print(f" index={index:3d} ==>  Value ={item:5f} ")
            index += 1
        return array
